root/url/utility/isGreyedOut.js

```diff
@@ -7,7 +7,12 @@
  * later version: http://www.gnu.org/licenses/gpl-2.0.txt
  */
 
-const greyRegExp = new RegExp('^https?://(?:www\.)?decoda\.com/');
+/*
+ * NB: No URLs currently need to be greyed out;
+ * this should always return false until that changes
+ */
+
+const greyRegExp = new RegExp('/(?!)/');
```

Comment:
Sorry, the `/(?!)/` was RegExp literal syntax

```suggestion
const greyRegExp = new RegExp('(?!)');
```